extensions/aws/processors/FetchS3Object.h

```diff
@@ -46,6 +46,7 @@ class FetchS3Object : public S3Processor {
       .supportsExpressionLanguage(true)
       .build();
   EXTENSIONAPI static constexpr auto Version = core::PropertyDefinitionBuilder<>::createProperty("Version")
+      .withDescription("The Version of the Object to download")
```

Comment:
duplicated line

Reply:
good catch fixed it in https://github.com/apache/nifi-minifi-cpp/pull/1707/commits/d64e7f890b82be04ec5a75bc484a1ebcbb001f32